scipy/stats/_continuous_distns.py

```diff
@@ -9151,6 +9151,8 @@ class rv_histogram(rv_continuous):
       The second containing the (n+1) bin boundaries
       In particular the return value np.histogram is accepted
 
+        .. versionadded:: 1.10.0
+
```

Comment:
```suggestion
```
This was included in the PR gh-16768 that is being backported, but it is unrelated and should not be packported.

Reply:
included